.github/ISSUE_TEMPLATE/releases/release_checklist.md

```diff
@@ -0,0 +1,43 @@
+# .NET Core Release Checklist
+
+## Release Versions
+
+_The set of .NET Core versions that are being released as a unit._
+
+* 2.1.X: <runtime/SDK>
+* 3.1.X: <runtime/SDK>
+
+
+
+1. - [ ] Incase of 2.x, run auto-update(refer docs). Incase of 3.X:
+      - [ ] if maestro auto-PR is active, then verify SHA1s in version.Details with manifest versions in VSU share dir (refer docs), make sure that we match the versions
+      - [ ] in-case auto-PR updates are unavailable, checkout a local branch for the pertinent version and run darc updates(refer docs). 
+          - [ ] Push this local branch upstream and start the release PR for the N version that is to be released.
+1. - [ ] Change the `PrivateSourceBuiltArtifactsPackageVersion` in `eng/Versions.props` to match N-1 release artifacts
+1. - [ ] Make sure that `source-build/ProdConFeed.txt` contains the latest feed required for the release. For info on latest feed (refer docs)
+1. - [ ] Verify that we are building with the appropriate stability and release tag (either none for RTM, or "preview1" etc).  This can apply to the SDK itself as well as the core-setup runtime component
```

Comment:
Does this mean we should do a local build? Where should we find `dotnet` to run these commands?

Reply:
Right. I'll clarify that.